doc/commands/bbgo_completion.md

```diff
@@ -0,0 +1,70 @@
+# bbgo Completion
```

Comment:
Cool!!

Is this hand-written or automatically generated doc?

if it's hand-written, please move it to doc/topics, thanks!

Reply:
it's hand-written after I make bbgo.

Okey! I will move it to doc/topics.

Reply:
I have moved it to doc/topics.